docs/docs/examples/database/AMAZON_RDS.md

```diff
@@ -0,0 +1,25 @@
+# Creating an RDS database instance
+This example provides instructions for creating an Amazon [RDS DB instance](https://aws.amazon.com/rds/){.external}.
+
+## Prerequisites
+* An `AWS account`, `IAM user`, `VPC` (or [default VPC](https://docs.aws.amazon.com/vpc/latest/userguide/default-vpc.html){.external}) and `security group` are required before an RDS DB instance can be created. See [Setting up for Amazon RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_SettingUp.html){.external} for further instructions.
+
+## Database creation
+There are two steps for creating the database:
+
+### 1. Initialize database server
+For details on standing up an RDS DB server follow the guide: [Creating an Amazon RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html){.external}. 
+### 2. Create database
+For details on creating the database user and database itself follow the appropriate guide below:
+
+=== "Jira"
```

Comment:
Example of tabbed content. This is very useful when we have product variants.

https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#usage